libs/service-portal/applications/src/screens/ApplicationList/ApplicationList.tsx

```diff
@@ -79,12 +81,14 @@ const ApplicationList: ServicePortalModuleComponent = () => {
 
           return (
             <ActionCard
-              key={application.id}
-              date={format(new Date(application.modified), dateFormat)}
+              key={`${application.id}-${index}`}
+              date={format(new Date(application.modified), formattedDate)}
               heading={application.name || application.typeId}
               tag={{
                 label: isComplete
                   ? formatMessage(m.cardStatusDone)
+                  : isOpen
+                  ? formatMessage(m.cardStatusOpen)
                   : formatMessage(m.cardStatusInProgress),
```

Comment:
I believe the designers want to have some generic messages for every application and not the custom for each application so the user can have common knowledge when they consult its list of applications

Reply:
Makes sense. Though I think it should be very obvious that the application is "Waiting for employer approval".

Maybe we can show that as a point on the progress instead?

Reply:
I discussed about it with Jóhanna and she doesn't want to create confusion by showing another status than the generic ones ("completed", "rejected", and "in progress").

However, since we are likely going to show a detailed page for each application showing the different steps the application contains and the status of each of them, the `stateTitle` will be of good use there